internal/controller/core/configmap_controller_test.go

```diff
@@ -13,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package controllers
+package core
```

Comment:
needed by kubebuilder?

Reply:
Well, this is what you get when using kubebuilder to setup the code:

`kubebuilder create api --group core --version v1 --kind ConfigMap --controller=true --resource=false`

I'd say this is the intended way. Apparently, in the past, we have been doing it manually without using `kubebuilder`.

Reply:
that's nice :)